apps/etterlatte-saksbehandling-ui/client/src/components/behandling/Behandling.tsx

```diff
@@ -34,13 +34,15 @@ export const Behandling = () => {
     }
   }, [behandlingIdFraURL, behandling?.id])
 
-  return (
-    <>
-      {behandling?.søker && <PdlPersonStatusBar person={behandling?.søker} />}
-      {behandling && <StegMeny behandling={behandling} />}
-
-      {isPending(fetchBehandlingStatus) && <Spinner label="Henter behandling ..." visible />}
-      {isSuccess(fetchBehandlingStatus) && (
+  return mapAllApiResult(
+    fetchBehandlingStatus,
+    <Spinner label="Henter behandling ..." visible />,
+    <Spinner label="Henter behandling ..." visible />,
```

Comment:
Når både initial og pending returnerer det samme kunne du like så godt brukt `mapApiResult` som kombinerer de to.

Reply:
Ja det er sant, var litt usikker på initial burde være her, null eller spinner?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/2345/commits/ce15c683c5058c2430023b39d4f9b755d29d8353

Reply:
det ble null, vet jo ikke om det feiler...